content-services/7.4/support/index.md

```diff
@@ -82,6 +82,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Alfresco Federation Services 3.0 | |
 | Identity Service 2.0 | |
 | Identity Service 1.8 | |
+| Keycloak 21.1.2 | |

```

Comment:
@vprovaggi Any objection to moving **Keycloak 21.1.2** above IDS 2.0 in both pages?

Reply:
@anxumalo no objection, do as you think should work best